reporter/parca_reporter.go

```diff
@@ -182,15 +216,16 @@ func (r *ParcaReporter) addMetadataForPID(pid util.PID, lb *labels.Builder) {
 	}
 }
 
-func (r *ParcaReporter) labelsForPID(pid util.PID, comm string) labelRetrievalResult {
-	if labels, exists := r.labels.Get(pid); exists {
+func (r *ParcaReporter) labelsForKey(key labelsKey) labelRetrievalResult {
+	if labels, exists := r.labels.Get(key); exists {
 		return labels
 	}
 
 	lb := &labels.Builder{}
 	lb.Set("node", r.nodeName)
-	lb.Set("comm", comm)
-	r.addMetadataForPID(pid, lb)
+	lb.Set("thread_name", key.comm)
+	lb.Set("tid", fmt.Sprint(key.tid))
```

Comment:
If we make these on by default as opposed to opt-in via relabeling, then we should probably also have the `pid` by default.

Also, can you add documentation about these here: https://github.com/parca-dev/parca-agent?tab=readme-ov-file#metadata-labels